documents/plan.md

```diff
@@ -1,11 +1,16 @@
 # Project plan
 
 ## Language evaluation
+I'll know that my language has accomplished its goals when it is possible to get from it any output that is even possible using the back end I have in mind. It should also take less time to take already-existing tab and re-write it in this language than it takes to re-write it in a graphical tab editor. Ideally, it should require fewer characters in this language to invoke certain behaviors (e.g. tripleted notes) than the back end requires.
 
 ## Implementation plan
+I've already found a fully-functioning back end, so I should spend my entire time working on a parser and plugging that in to the back end. As such I think I'll spend as long as I feel is necessary on language design, which will probably include some tweaking to make it play nice with the back end. If I meet all my goals as stated in the language evaluation section above and still have time to spare, I'll try adding an additional type of output that the user can request when a
```

Comment:
It would be good to have more specificity than "as long as I feel necessary". Vague deadlines such as these are how projects get off-track, and the purpose of this assignment was to set yourself up for success by being as detailed as possible.